jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,238 @@
+    "file_info = file_infos[0]\n",
+    "\n",
+    "test_file_name = get_property(file_info, \"name\")\n",
+    "program_name = get_property(file_info, \"Test~Procedure\") or get_property(file_info, file_name_prop)\n",
+    "\n",
+    "group_names = []\n",
+    "channels = []\n",
+    "formatted_properties = []\n",
+    "\n",
+    "for group in file_info.groups:\n",
+    "    group_names.append(group.name)\n",
+    "    channels.append(group.channels)\n",
+    "    max_values_of_group = []    \n",
+    "    min_values_of_group = []    \n",
+    "    mean_values_of_group = []    \n",
+    "\n",
+    "    for channel in group.channels:\n",
+    "        minimum = float(get_property(channel, \"minimum\") or \"NaN\")\n",
+    "        maximum = float(get_property(channel, \"maximum\") or \"NaN\")\n",
+    "        mean_values_of_group.append((minimum + maximum) / 2)\n",
+    "        max_values_of_group.append(maximum)\n",
+    "        min_values_of_group.append(minimum)\n",
+    "\n",
+    "        \n",
+    "    # Calculate statistic values from metadata \n",
+    "    abs_min = max(max_values_of_group)\n",
+    "    abs_max = min(max_values_of_group)\n",
+    "    abs_mean = sum(mean_values_of_group) / float(len(mean_values_of_group))\n",
+    "    formatted_properties.append(\"Absolute Maximum: \"+str(round(abs_max,3))+\" °C\"+\n",
+    "                                \",Absolute Minimum: \"+str(round(abs_min,3))+\" °C\"+\n",
+    "                                \",Mean Value: \"+str(round(abs_mean,3))+\" °C\") "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "papermill": {
+     "parameters": {
+      "comment_grp_1": "Checked",
+      "show_channel_of_group": 2
+     }
+    },
+    "tags": [
+     "Parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# The Analysis Automation Procedure Package parameter cell corresponds to the ANP parameter definition. \n",
+    "# (see: cell \"property inspector\" => area: Advanced Tools => cell metadata)\n",
+    "\n",
+    "comment_grp_1 = \"Checked\"\n",
+    "show_channel_of_group = 2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The  Analysis Automation Procedure Package parameter set is available from the cell below the parameter cell itself \n",
+    "\n",
+    "# Enter the parameter in the info box of the plot. \n",
+    "formatted_properties[1] +=\",Parameter: \" + str(comment_grp_1)\n",
+    "formatted_properties[0] +=\",Channel #: \" + str(show_channel_of_group)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get bulk data from DataLink\n",
+    "bulk_data = []\n",
+    "file_id = data_links[0]['fileId']\n",
+    "grp_idx = 0\n",
+    "\n",
+    "for group in file_info.groups:\n",
+    "    chn_specs = []\n",
+    "    grp_idx += 1 \n",
+    "    chn_idx = 0\n",
```

Comment:
The variables `grp_idx` and `chn_idx` are not used and can be removed.

Reply:
ok